### PR TITLE
Create archiver_ubi_8.4.sh

### PR DESCRIPTION
**Pull request by nageshlop**
Verified build and test of archiver package on ppc64le with master (v3.5.0) and v3.2.0 as updated in the spread sheet.

### CODE REVIEW

**Comment by gururajrkatti** [comment]
@ezeeyahoo can you pl. review

**Comment by nageshlop** [comment]
Hi @ezeeyahoo, Could you let me know if there are any changes required or good to proceed?

**Comment by Siddhesh-Ghadi** [comment]
Script looks good to me. Can we merge it? @kandarpamalipeddi

File: a/archiver/archiver_ubi_8.4.sh [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+
+if [ -z "$1" ]; then
+  export VERSION=master
+else
+  export VERSION=$1
+fi
```
Comment (kandarpamalipeddi) [inline a/archiver/archiver_ubi_8.4.sh:24]: Please use either `v3.5.0` or `v3.2.0` instead of master.

File: a/archiver/archiver_ubi_8.4.sh [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+if [ -d "archiver" ] ; then
+  rm -rf archiver
+fi
+
+# Dependency installation
+sudo dnf update
+sudo dnf install -y go git wget
+
+# Download the repos
+git clone https://github.com/mholt/archiver
```
Comment (kandarpamalipeddi) [inline a/archiver/archiver_ubi_8.4.sh:35]: It would be really good, if this is cloned to GOPATH.

**Review by kandarpamalipeddi — CHANGES_REQUESTED** [review]

**Pull request closed without merging**